### PR TITLE
[ISSUE #3963] Method invocation 'getBytes' may produce 'NullPointerException'.

### LINKED ISSUE #3963: [Enhancement] Method invocation 'getBytes' may produce 'NullPointerException'.
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Enhancement Request

image
located at:
eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/admin/handler/QueryWebHookConfigByIdHandler.java line 63
eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/admin/handler/QueryWebHookConfigByManufacturerHandler.java line 76

### Describe the solution you'd like

Use `Objects.requireNonNull()` to avoid NPE.

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by ZahaanMahajan**
Fixes #3963 

Method invocation 'getBytes' may produce 'NullPointerException'. 

In the modified code, Objects.requireNonNull() is applied to the return value of JsonUtils.toJSONString(result). If the result is null, a NullPointerException will be thrown immediately, preventing the subsequent invocation of getBytes().

By using Objects.requireNonNull(), it explicitly check for nullness and handle it in a more controlled manner, ensuring that it don't encounter unexpected null values during the execution of the code.

### CODE REVIEW

**Review by mxsm — DISMISSED** [review]
LGTM~

**Review by Alonexc — DISMISSED** [review]
LGTM

File: eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/admin/handler/QueryWebHookConfigByIdHandler.java [hunk 1]
```diff
@@ -60,7 +61,7 @@ public void handle(HttpExchange httpExchange) throws IOException {
 
         try (OutputStream out = httpExchange.getResponseBody()) {
             WebHookConfig result = operation.queryWebHookConfigById(webHookConfig); // operating result
-            out.write(JsonUtils.toJSONString(result).getBytes(Constants.DEFAULT_CHARSET));
+            out.write(Objects.requireNonNull(JsonUtils.toJSONString(result).getBytes(Constants.DEFAULT_CHARSET)));
```
Comment (Alonexc) [inline eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/admin/handler/QueryWebHookConfigByIdHandler.java:64]: I think it should be done as follows:
`out.write(Objects.requireNonNull(JsonUtils.toJSONString(result)).getBytes(Constants.DEFAULT_CHARSET));`

File: eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/admin/handler/QueryWebHookConfigByManufacturerHandler.java [hunk 2]
```diff
@@ -73,7 +73,7 @@ public void handle(HttpExchange httpExchange) throws IOException {
 
         try (OutputStream out = httpExchange.getResponseBody()) {
             List<WebHookConfig> result = operation.queryWebHookConfigByManufacturer(webHookConfig, pageNum, pageSize); // operating result
-            out.write(JsonUtils.toJSONString(result).getBytes(Constants.DEFAULT_CHARSET));
+            out.write(Objects.requireNonNull(JsonUtils.toJSONString(result).getBytes(Constants.DEFAULT_CHARSET)));
```
Comment (Alonexc) [inline eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/admin/handler/QueryWebHookConfigByManufacturerHandler.java:76]: same as above.

**Comment by xwm1992** [comment]
@ZahaanMahajan you can merge the latest master branch code to fix the ci check error.

**Comment by ZahaanMahajan** [comment]
How to do that?

**Comment by xwm1992** [comment]
> I didn't get you. How to do that?

pull apache:master to your local branch and push it to your remote master branch.

**Review by xwm1992 — APPROVED** [review]

**Review by Alonexc — APPROVED** [review]

**Review by lrhkobe — APPROVED** [review]

**Review by mxsm — APPROVED** [review]

**Pull request merged**